pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -128,14 +128,14 @@ const (
 )
 
 const (
-	// The amount of time the nodecontroller should sleep between retrying NodeStatus updates
+	// The amount of time the nodecontroller should sleep between retrying node health updates
 	retrySleepTime = 20 * time.Millisecond
 )
 
-type nodeStatusData struct {
+type nodeHealthData struct {
 	probeTimestamp           metav1.Time
 	readyTransitionTimestamp metav1.Time
-	status                   v1.NodeStatus
+	status                   *v1.NodeStatus
```

Comment:
The commit message is slightly misleading since you also changed this to a pointer.

Reply:
Updated the commit message to include the pointer change now. :)